docs/components.md

```diff
@@ -21,62 +21,83 @@ We are using [lerna.js](https://lernajs.io/) to manage our packages and publish
 
 Lerna is a tool that optimizes the workflow around managing multi-package repositories with git and npm.
 
+It manages dependencies between our components so that if the version number of `globals` is bumped, all components dependent on `globals` will also have their version bumped.
+
+To do: We need to do more work to clarify how versioning, including managing breaking changes, will work.
+
 ### Publishing components manually (while in Alpha)
 
 In a new branch:
 
-- Run the `npm run build:packages` task to ensure any changes to `src/components/component-name` are copied to `packages/component-name`.
+1. Run the `npm run build:packages` task.
+
+2. Ensure all changes to `src/components/component-name` have been copied to `packages/component-name`. (Atom show changed directories in green).
```

Comment:
Nit: Could we use `git status` instead of 'atom' since not everyone will be using atom as their text editor?

Reply:
Fair point, it was too specific to my particular set up. I took it out 👍